src/logic/schedule-logic/month-info.logic.ts

```diff
@@ -23,6 +23,14 @@ export class MonthInfoLogic {
     return this._verboseDates.map((d) => d.date);
   }
 
+  public get holidays(): number[] {
+    const holidays: number[] = [];
+    this._verboseDates.forEach((d) => {
+      d.isPublicHoliday && holidays.push(d.date);
+    });
```

Comment:
`const holidays = this._verboseDates.filter(d => d.isPublicHoliday).map((d, index => index)`
Zwróć uwagę, na to że na końcu jest indeks